### PR TITLE
ed

### PR DESCRIPTION
**Pull request by azime**
Create physical mode of GtfsData

### CODE REVIEW

**Comment by l-vincent-l** [comment]
jenkins, test this please

File: source/ed/connectors/fusio_parser.cpp [hunk 1]
```diff
@@ -973,8 +973,8 @@ ed::types::CommercialMode* GtfsData::get_or_create_default_commercial_mode(Data
 }
 
 ed::types::PhysicalMode* GtfsData::get_or_create_default_physical_mode(Data & data) {
-    if (! default_physical_mode) {
-        ed::types::PhysicalMode* default_physical_mode = new ed::types::PhysicalMode();
+    if (default_physical_mode == nullptr) {
```
Comment (antoine-de) [inline source/ed/connectors/fusio_parser.cpp]: why did you change the test ? you prefer the '== nullptr' syntax ?

**Comment by azime** [comment]
correction
